nntrainer/layers/depthwise_conv2d_layer.h

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 Donghak Park <donghak.park@samsung.com>
+ *
+ * @file   depthwise_conv2d_layer.h
+ * @date   27 March 2024
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Donghak Park <donghak.park@samsung.com>
+ * @bug    No known bugs except for NYI items
+ * @brief  This is Depthwise Convolution Layer Class for Neural Network
+ *
+ */
+
+#ifndef __DEPTHWISE_CONV2D_LAYER_H_
+#define __DEPTHWISE_CONV2D_LAYER_H_
+#ifdef __cplusplus
+
+#include <memory.h>
+
+#include <common_properties.h>
+#include <layer_impl.h>
+
+namespace nntrainer {
+
+constexpr const unsigned int DEPTHWISE_CONV2D_DIM = 2;
```

Comment:
It'd be better if this is a member of DepthwiseConv2DLayer class.

Reference: https://www.geeksforgeeks.org/software-engineering-coupling-and-cohesion/